source/models/validation-errors.md

```diff
@@ -0,0 +1,71 @@
+Displaying validation errors from a backend server is a common pattern
+when attempint to update records based on user feedback.
+
+Every DS.Model has an `errors` property. This can be used to display
+validation error messages returned from the server when a
+`record.save()` rejects.  This works automatically with
+`DS.ActiveModelAdapter`, but you can implement
```

Comment:
Unfortunately no, the current RESTAdapter doesn't return an InvalidError, although I believe it did that in the past (~1 year ago).  https://github.com/emberjs/data/blob/master/packages/ember-data/lib/adapters/rest-adapter.js#L718-L733